src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -3,28 +3,6 @@
 # This file defines the default (English) variants of all of the internationalised messages
 ascanalpha.desc = Active Scan Rules - alpha
 
-ascanalpha.examplefile.name=An example active scan rule which loads data from a file
```

Comment:
I can give it a shot, problem is there is also a merge involved.  Probably be easier to get the original file and copy and paste the 4 lines.